pkg/controllers/deprovisioning/types.go

```diff
@@ -100,10 +100,10 @@ func (a action) String() string {
 }
 
 type Command struct {
-	nodesToRemove   []*v1.Node
-	action          action
-	replacementNode *scheduling.Node
-	created         time.Time
+	nodesToRemove    []*v1.Node
+	action           action
+	replacementNodes []*scheduling.Node
+	createdAt        time.Time
 }
 
 func (o Command) String() string {
```

Comment:
Musing: There's some stuff in here that would be better put with structured logging. Maybe there's room for something like `WithCommand` that injects these structured details into the logger rather than putting them in the log message?